.husky/precommit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn lint
```

Comment:
this is good but not _quite right_, this works in Peer Server because `yarn lint` is set up differently, so that it only lints staged files. I would recommend adding a new command like `yarn lint-staged` or something which matches `yarn lint` in the server repo.